README.md

```diff
@@ -1,13 +1,34 @@
-# Testing
+# Contributing
+
+## Requirements
+
+- Python 3
+- [go-task](https://taskfile.dev/installation)
```

Comment:
```suggestion
* Python 3.10 or higher
* [Task] 3.38 or higher
```

We haven't set up a linting task for it yet, but we do have a [markdownlint config](https://github.com/y-scope/yscope-dev-utils/blob/main/exports/lint-configs/.markdownlint-cli2.yaml).